tests/feature/feature_console/zypper_crit_sec_fix_only.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's feature tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distbution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Feature #318760: Update critical security fixes only
+# Test Case #1480288: Test Feature: Update critical security fixes only
+
+use base "consoletest";
+use testapi;
+use registration;
+
+sub run() {
+    select_console 'root-console';
+
+    my $not_registered = qr/"status":"Not Registered"/;
+    my $registered     = qr/"status":"Registered"/;
+
+    if (check_var 'DISTRI', 'sle') {
+        script_run "SUSEConnect --status | tee /dev/$serialdev", 0;
+        my $out = wait_serial [$not_registered, $registered];
+        if ($out =~ $not_registered) {
+            yast_scc_registration;
+        }
+    }
```

Comment:
when you are automatically registering the system from within this test module this can have a very big impact on all tests. I think the test should not be that "intelligent" but rely on the system being registered already, either by the separate scc_registration step as is used in some tests or during installation.